app/views/404.hbs

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Not found - Streetmix</title>
+
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Language" content="en">
+    <meta name="description" content="Design, remix, and share your neighborhood street – all in your browser! Add trees or bike paths, widen sidewalks or traffic lanes, learn how your decisions can impact your community.">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="transparent">
+
+    <link href="/favicon.ico" rel="shortcut icon">
+    <link href="/assets/css/styles.css" rel="stylesheet">
+  </head>
+  <body tabindex="1">
+    <div id="error">
+      <div class="clouds-background">
+        <div class="rear-clouds"></div>
+        <div class="front-clouds"></div>
+      </div>
+      <div class="error-content">
+        <div class="streetmix-logo"></div>        
```

Comment:
I think we'll also have to update this line in this other file https://github.com/streetmix/streetmix/blob/4faf5cb800e138463389ca500de9b97fdd889c3c/assets/css/_chrome.scss#L78 to make sure that `.streetmix-logo` picks up the same styling as one inside of a `#loading` element.

Reply:
There's a `.streetmix-logo` in the menu bar that also has its own styling and the `#loading` forces this to inherit from that one. It might not be a super great way of doing this so feel free to propose another way to name / organize these styles if you think that's necessary.

Reply:
Apparently the contents in `#loading` is quite different from `#error`. Forcing the logo in error  to inherit the same styling as the one loading causes the logo to disappear. Trying to rewrite a new styling for the logo in error.